_posts/2022-08-14-aug-update.markdown

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "Managarm: August 2022 Update"
+excerpt:
+---
+<span style="font-size: 11pt;">Post by
+Arsen Arsenović ([@ArsenArsen](https://github.com/ArsenArsen)),
+Dennis Bonke ([@Dennisbonke](https://github.com/Dennisbonke)),
+Geert Custers ([@geertiebear](https://github.com/geertiebear)),
+Alexander van der Grinten ([@avdgrinten](https://github.com/avdgrinten)),
+Matt Taylor ([@64](https://github.com/64))
```

Comment:
Are we mentioning Leo here? :smile:

Reply:
Originally it was only the people that worked on the post, tho that was months ago. Not against adding Leo to the list.